tests/laser_STC.SI.1Rank.sh

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env bash
+
+# Copyright 2024
+#
+# This file is part of HiPACE++.
+#
+# Authors: Xingjian Hui
+# License: BSD-3-Clause-LBNL
+
+
+# This file is part of the HiPACE++ test suite.
+# It initializes a Hipace simulation of a gaussian laser with initial STC parameters
+# and test the correctness of the STC factors
+
+# Abort on first encountered error
+set -eu -o pipefail
+
+# Read input parameters
+HIPACE_EXECUTABLE=$1
+HIPACE_SOURCE_DIR=$2
+
+HIPACE_EXAMPLE_DIR=${HIPACE_SOURCE_DIR}/examples/laser
+HIPACE_TEST_DIR=${HIPACE_SOURCE_DIR}/tests
+
+FILE_NAME=`basename "$0"`
+TEST_NAME="${FILE_NAME%.*}"
+
+# Run the simulation with initial phi2
+mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
+        laser.phi2 = 2.4e-26 \
+        laser.STC_theta_xy = 0 \
+        hipace.file_prefix = $TEST_NAME
+# Compare the result with theory
+$HIPACE_EXAMPLE_DIR/analysis_laser_init_chirp.py --output-dir=$TEST_NAME \
+        --chirp_type="phi2"
+
+rm -rf $TEST_NAME
+
+# Run the simulation with initial zeta
+mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
+        laser.zeta = 2.4e-19 \
+        laser.STC_theta_xy = 0 \
+        hipace.file_prefix = $TEST_NAME
+
+# Compare the result with theory
+$HIPACE_EXAMPLE_DIR/analysis_laser_init_chirp.py --output-dir=$TEST_NAME \
+        --chirp_type="zeta"
+
+rm -rf $TEST_NAME
+
+mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
+        laser.beta = 2e-17 \
+        laser.STC_theta_xy = 0 \
+        hipace.file_prefix = $TEST_NAME
```

Comment:
Let's make `laser.STC_theta_xy = 0` default and not specify it every time.
Furthermore, could you run 1 single simulation with all chirps?
```
        laser.phi2 = 2.4e-26 \
        laser.zeta = 2.4e-19 \
        laser.beta = 2e-17 \
```
Would your post-processing disentangle these separately? That would be neat and quite convincing on the robustness of post-processing.

```suggestion
mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
        laser.phi2 = 2.4e-26 \
        hipace.file_prefix = $TEST_NAME
# Compare the result with theory
$HIPACE_EXAMPLE_DIR/analysis_laser_init_chirp.py --output-dir=$TEST_NAME \
        --chirp_type="phi2"

rm -rf $TEST_NAME

# Run the simulation with initial zeta
mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
        laser.zeta = 2.4e-19 \
        hipace.file_prefix = $TEST_NAME

# Compare the result with theory
$HIPACE_EXAMPLE_DIR/analysis_laser_init_chirp.py --output-dir=$TEST_NAME \
        --chirp_type="zeta"

rm -rf $TEST_NAME

mpiexec -n 1 $HIPACE_EXECUTABLE $HIPACE_EXAMPLE_DIR/inputs_chirp \
        laser.beta = 2e-17 \
        hipace.file_prefix = $TEST_NAME
```

Reply:
@huixingjian. I think you mention that the post-processing currently would not support this, correct? If so, it is alright to keep as is and reconsider once we have a better understanding on this.